app/controllers/SessionsController.scala

```diff
@@ -410,4 +427,11 @@ class SessionsController @Inject()(messagesApi: MessagesApi,
       .flashing("message" -> "Feedback form schedule initiated"))
   }
 
+  def shareContent(id: String): Action[AnyContent] = action.async { implicit request =>
+    val futureSessionOption: Future[Option[SessionInfo]] = sessionsRepository.getById(id)

```

Comment:
Please rename this variable name to eventualMaybeSession as this is what we have been following elsewhere.
maybe for Option
eventual for Future